apps/src/templates/rubrics/AiAssessmentFeedback.jsx

```diff
@@ -30,21 +31,38 @@ export default function AiAssessmentFeedback({learningGoalKey}) {
   const receivedTimer = useRef();
   const disappearAfter = 20000;
 
+  const baseUrl = '/learning_goal_ai_evaluation_feedbacks';
+
   const radioAiFeedbackCallback = radioButtonData => {
     setAIFeedback(radioButtonData);
+    if (radioButtonData === thumbsupval) {
+      const bodyData = JSON.stringify({
+        learningGoalAiEvaluationId: aiInfo.id,
+        aiFeedbackApproval: thumbsupval,
+      });
+      HttpClient.post(`${baseUrl}`, bodyData, true, {
+        'Content-Type': 'application/json',
+      }).catch(error => {
+        console.log(error);
+      });
+    }
   };
 
   const submitAiFeedbackCallback = () => {
     const bodyData = JSON.stringify({
-      learningGoalKey: learningGoalKey,
-      aiFalseNeg: aiFalseNeg,
-      aiFalsePos: aiFalsePos,
-      aiVague: aiVague,
-      aiOther: aiFeedbackOther,
-      aiOtherContent: aiOtherContent,
+      learningGoalAiEvaluationId: aiInfo.id,
+      aiFeedbackApproval: aiFeedback,
+      falsePositive: aiFalsePos,
+      falseNegative: aiFalseNeg,
+      Vague: aiVague,
+      feedbackOther: aiFeedbackOther,
+      otherContent: aiOtherContent,
+    });
+    HttpClient.post(`${baseUrl}`, bodyData, true, {
+      'Content-Type': 'application/json',
+    }).catch(error => {
+      console.log(error);
```

Comment:
per previous comment, please let js error propagate and do not catch it here, unless you are going to catch it and prompt the user to try again